pkg/bmc/access_test.go

```diff
@@ -1156,8 +1156,8 @@ func TestUnknownType(t *testing.T) {
 }
 
 func TestBuildBIOSCleanSteps(t *testing.T) {
-	var True bool = true
-	var False bool = false
+	var True = true
+	var False = false
```

Comment:
I think, it is a good time to remove these redundant assignments.

Reply:
+1